deployment/computer-vision/local-images.ipynb

```diff
@@ -0,0 +1,440 @@
+    "    \"ymin\": 0,\n",
+    "    \"xmin\": 0,\n",
+    "    \"ymax\": 0,\n",
+    "    \"xmax\": 0\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_version = registered_model.create_standard_model(\n",
+    "    model_cls = DetectObject,\n",
+    "    environment = Python(requirements = ['tensorflow', 'tensorflow_hub', 'matplotlib']),\n",
+    "    model_api = ModelAPI([input], [output]),\n",
+    "    name = 'v1'\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deploy Model to Endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = client.get_or_create_endpoint('object-detection')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint.update(model_version, wait = True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "deployed_model = endpoint.get_deployed_model()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download Images"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mkdir images"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "urls = [\n",
+    "    'http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_0.jpg',\n",
+    "    'http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_1.jpg',\n",
+    "    'http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_2.jpg',\n",
+    "    'http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_3.jpg',\n",
+    "    'http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_4.jpg',\n",
+    "    'http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_5.jpg'\n",
+    "]\n",
+    "\n",
+    "for url in urls:\n",
+    "    urllib.request.urlretrieve(url, f\"images/{url.split('/')[-1]}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "folder = 'images/'\n",
+    "results = []\n",
+    "\n",
+    "for root, dirs, files in os.walk(folder):    \n",
+    "    for file in sorted(files):\n",
+    "        print(f\"Processing {file}...\")\n",
+    "        img = Image.open(f\"{root}/{file}\")\n",
+    "        img_arr = np.array(img).tolist()\n",
+    "        result = deployed_model.predict([file, img_arr])\n",
+    "        results.append(result)\n",
+    "        break"
```

Comment:
Do we want this `break` in the final notebook, or was this just for testing?

Reply:
Yes. At this moment, we just want to add to `results` the first entity found.